### ISSUE TITLE
feature: Adding additional socials

### ISSUE THREAD

**Issue by jackh-sh** [JackHumphries9/Socially#2]
Hi,
   Please create a new issue for any new socials you would like to add.

Currently I have these planned:
- Teams
- [Reddit](https://github.com/JackHumphries9/Socially/issues/29)
- Twitter

**Comment by jackh-sh** [comment]
Added Slack and LinkedIn.

**Comment by Biggy1606** [comment]
Hi, will be possible to add **Signal**?
https://github.com/signalapp/Signal-Desktop

**Comment by jackh-sh** [comment]
Yeah I am just going to create a new issue for it so that I can remember to do it. I'll probably end up replacing Twitter with it.

**Comment by jackh-sh** [comment]
Please create a feature request for new socials.
